File: CLion/Day2W.cpp
//
// Created by poloj on 12-06-2026.
//
#include<stdio.h>
int main(void) {
    setbuf(stdout,0);
    char paragraph[100]="Bob hit an bat";
    char words[10][10];
    int wordCount=0;
    for (int i=0,j=0,index=0;i<sizeof(paragraph)-1;i++) {
        while (paragraph[i]!=32 && i<sizeof(paragraph)-1) {
            words[index][j]=paragraph[i];
            i=i+1;
            j=j+1;
        }
        wordCount=wordCount+1;
        words[index][i]='\0';
        index=index+1;
        j=0;
    }
    for (int i=0;i<wordCount;i++) {
        int j=0;
        while (words[i][j]!='\0') {
            printf("%c",words[i][j]);
            j++;
        }
        printf("\n");
    }
    return 0;
}

File: Java/Day2W.java
package Daywise;


public class Day2W {
    public static void main(String[] args){
        String paragraph = "Bob hit a ball Bob hit a ball";
        char [][]words= new char[10][100];
        int wordcount=0;
        for(int i=0,j=0,index=0;i<paragraph.length();i++){
            while(paragraph.charAt(i)!=32&& i<paragraph.length()-1){
                words[index][j]=paragraph.charAt(i);
                i=i+1;
                j=j+1;
            }
            wordcount=wordcount+1;
            words[index][j]='\0';
            index=index+1;
            j=0;
        }
        for(int i=0;i<wordcount;i++){
          for(int j=0;words[i][j]!='\0';j++) {
              System.out.print(words[i][j]);
          }
          System.out.println();
        }
    }
}
